### ISSUE TITLE
[Bug] XTTS output is non-deterministic

### ISSUE THREAD

**Issue by madey83** [idiap/coqui-ai-TTS#552]
### Describe the bug

Hello,

yesterday i installed latest version of the coqui according to documentation.

I've tried today do inferance for this text ( in Polish ) - "Sprawdzę, czy nikogo nie ma w środku i wtedy go spalę,"

generated voice had some quick long pauze after word "nikogo"

It does not matter if i've reduced temp from 0.75 to 0.50 or even 0.20. Each generation gives the same outcome.   

### To Reproduce

install following documentation

### Expected behavior

not adding pauze

### Logs

```shell

```

### Environment

```shell
python collect_env_info.py
{
    "CUDA": {
        "GPU": [
            "NVIDIA GeForce RTX 2060"
        ],
        "available": true,
        "version": "12.1"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "2.2.0+cu121",
        "TTS": "0.28.0.dev0",
        "numpy": "1.26.4"
    },
    "System": {
        "OS": "Windows",
        "architecture": [
            "64bit",
            "WindowsPE"
        ],
        "processor": "Intel64 Family 6 Model 165 Stepping 2, GenuineIntel",
        "python": "3.10.11",
        "version": "10.0.19045"
    }
}
```

**Comment by eginhard** [comment]
I assume you are talking about the XTTS model? If you don't set a fixed seed the output of XTTS is non-deterministic, i.e. it will be different each time. XTTS may also occasionally insert pauses at incorrect locations or hallucinate speech that was not in the input text. There is no way to fully prevent that.

Fixing the seed, the following code produces the exact same output in version 0.27.0 and the current `dev` branch (https://github.com/idiap/coqui-ai-TTS/commit/36c6ff800bb4b39956dce2fb044e109c855359c6):

```python
import torch
from TTS.api import TTS

xtts = TTS("tts_models/multilingual/multi-dataset/xtts_v2")
language = "pl"
text = "Sprawdzę, czy nikogo nie ma w środku i wtedy go spalę,"

torch.manual_seed(123)
xtts.tts_to_file(text, language=language, speaker="Ana Florence")
```

**Comment by madey83** [comment]
this is not solution. Pauze after word "nikogo" still there even if i add seed.

When i inferance the same text with the same settings in Alltalk_beta there is no issue with pauze at all.

**Comment by eginhard** [comment]
You can try with a different seed. With my code above there is no pause and this output hasn't changed in recent Coqui versions:
[output.mp3](https://github.com/user-attachments/files/24959605/output.mp3) (I noticed that Alltalk uses an older version of the XTTS model, so even if you set the same seed there you'll get a different output)

If you don't want random pauses or other hallucinations at all, you'll need to use another model, that's just how XTTS was trained.
